CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-20.4.2
+
+This release brings a number of CLI fixes and Controller improvements.
+
+* CLI
+  * Fixed a bug that caused the proxy to crash after upgrade if
+    `--skip-outbound-ports` or `--skip-inbound-ports` were used
```

Comment:
More precisely, the bug will crash the pods after they get injected, not just proxy :grimacing: